clang/test/Analysis/NewDeleteLeaks.cpp

```diff
@@ -14,6 +14,8 @@
 
 #include "Inputs/system-header-simulator-for-malloc.h"
 
+#include <utility>
```

Comment:
Could you extend some of the existing header simulators with the required stuff?We are not allowed to use  host compiler headers in these tests to keep things hermetic.

Reply:
Do you have any suggestions on how to do this? Should I just copy/paste the necessary functionality from utility into the file? What is standard practice?